service/src/main/java/bio/terra/catalog/service/SchemaService.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.catalog.service;
+
+import bio.terra.catalog.config.SchemaConfiguration;
+import bio.terra.common.exception.BadRequestException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.SpecVersion;
+import com.networknt.schema.ValidationMessage;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SchemaService {
+
+  private final JsonSchema schema;
+
+
+  @Autowired
+  public SchemaService(SchemaConfiguration schemaConfiguration) throws FileNotFoundException {
```

Comment:
I don't think an injected object should throw a checked exception. I would catch and re-throw this as a custom unchecked exception, such as `SchemaServiceInitializationException`.